code/modules/uplink/uplink_items.dm

```diff
@@ -959,7 +959,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	desc = "An upgraded version of the standard syndicate hardsuit. It features a built-in energy shielding system. \
 			The shields can handle up to three impacts within a short duration and will rapidly recharge while not under fire."
 	item = /obj/item/clothing/suit/space/hardsuit/shielded/syndi
-	cost = 30
+	cost = 35
```

Comment:
Y tho

Reply:
Its shield blocks 3 attacks, before needing to recharge. Only thing that pens shields are pulse shots